tox.ini

```diff
@@ -33,7 +33,7 @@ commands =
 skip_install = true
 usedevelop = false
 deps = flake8
-commands = flake8 --max-complexity=20 webgrid_ta webgrid_blazeweb_ta
+commands = flake8 --max-complexity=30 webgrid webgrid_ta webgrid_blazeweb_ta
```

Comment:
I certainly could. But, my goal in this release was simply to have webgrid code covered with flake8, as it was not before. I think I will leave this for another time, in #10